runtime/themes/dark_plus.toml

```diff
@@ -81,7 +81,6 @@
 
 "ui.virtual.whitespace" = { fg = "dark_gray" }
 "ui.virtual.ruler" = { bg = "borders" }
-"ui.virtual.indent-guide" = { bg = "dark_gray4" }
```

Comment:
This looks like an unrelated change? Although it does look better. Should it be

```toml
"ui.virtual.indent-guide" = { fg = "dark_gray4" }
```

instead?